### PR TITLE
Use specified DB name for preload file cache

### PR DESCRIPTION
**Pull request by burik666**

### CODE REVIEW

File: tools/file_packager.py [hunk 1]
```diff
@@ -568,7 +574,7 @@ def was_seen(name):
       var indexedDB = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
       var IDB_RO = "readonly";
       var IDB_RW = "readwrite";
-      var DB_NAME = 'EM_PRELOAD_CACHE';
+      var DB_NAME = "''' + indexeddb_name + '''";
```
Comment (kripken) [inline tools/file_packager.py]: are these quotes right? shouldn't it be `"'" + indexeddb_name + "'"` or `'\'' + indexeddb_name + '\''`?

File: tests/test_browser.py [hunk 2]
```diff
@@ -319,7 +319,7 @@ def make_main(path):
     ''')
 
     make_main('somefile.txt')
-    Popen([PYTHON, EMCC, os.path.join(self.get_dir(), 'main.cpp'), '--use-preload-cache', '--js-library', os.path.join(self.get_dir(), 'test.js'), '--preload-file', 'somefile.txt', '-o', 'page.html']).communicate()
+    Popen([PYTHON, EMCC, os.path.join(self.get_dir(), 'main.cpp'), '--use-preload-cache', '--indexedDB-name=test_db', '--js-library', os.path.join(self.get_dir(), 'test.js'), '--preload-file', 'somefile.txt', '-o', 'page.html']).communicate()
```
Comment (kripken) [inline tests/test_browser.py]: please add testing, instead of modifying an existing test. modifying means something that was tested before has been lost.

**Comment by kripken** [comment]
Also please add yourself to AUTHORS.

File: tools/file_packager.py (hunk 1 above)
Comment (burik666) [inline tools/file_packager.py]: Quotes are correct. 
It's Python's triple-quotes:
```
    code += r'''
      var indexedDB = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
      var IDB_RO = "readonly";
      var IDB_RW = "readwrite";
      var DB_NAME = "''' + indexeddb_name + '''";
      var DB_VERSION = 1;
```

Comment (kripken) [inline tools/file_packager.py]: oh, right, thanks.

**Comment by burik666** [comment]
it's ok?

**Comment by kripken** [comment]
Overall looks good, but I have a concern about the test. It runs the html twice, and relies on data being cached from the first run to the second, if I understand it correctly? But then why happens when you run the test twice in a row, in the second run, data was already cached? Perhaps it needs to be cleared.

Sorry for not noticing the new commits here, btw (github doesn't send emails on new commits, only comments, and I forgot to look).

**Comment by burik666** [comment]
if run the test twice, metadata.package_uuid will be changed and data reloaded.

**Comment by kripken** [comment]
Oh ok, great.

**Pull request merged**